### PR TITLE
[OneExplorer] Use TreeView API

### PR DESCRIPTION
**Pull request by dayo09**
This commit uses treeview api to exploit vscode's latest support.
This commit includes UI change that enables multi-selection.

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

### CODE REVIEW

**Comment by dayo09** [comment]
## GIF

0609-multiselection

This enables multi-selection of tree entries.

File: src/OneExplorer/OneExplorer.ts [hunk 1]
```diff
@@ -465,20 +465,26 @@ input_path=${modelName}.${extName}
...
-    context.subscriptions.push(
-        vscode.window.registerTreeDataProvider('OneExplorerView', oneTreeDataProvider));
 
-    const subscribeCommands = (disposals: vscode.Disposable[]) => {
+    // TODO Remove duplicated collapseAll command
+    this.treeView = vscode.window.createTreeView(
+        'OneExplorerView',
+        {treeDataProvider: oneTreeDataProvider, showCollapseAll: true, canSelectMany: true});
+
+
```
Comment (dayo09) [inline src/OneExplorer/OneExplorer.ts:479]:
```suggestion
```

**Comment by seanshpark** [comment]
Q) why is multiple treeview item selection is needed?

**Comment by dayo09** [comment]
@seanshpark Not now. If it support drag and drop or so, it will be in use. This is to use TreeView's `reveal` API, which reveals an tree entry in the view (expansion, selection, ...).

**Comment by seanshpark** [comment]
@dayo09 , OK thanks for the explanation.
I understand this as
- this PR will replace existing (whatever) to new TreeView (? not sure about the term)
- GIF image presented is an example of new TreeView feature
- API: https://code.visualstudio.com/api/extension-guides/tree-view

File: src/OneExplorer/OneExplorer.ts [hunk 2]
```diff
@@ -465,20 +465,25 @@ input_path=${modelName}.${extName}
...
+  public treeView: vscode.TreeView<OneNode|undefined>|undefined;
 
   constructor(context: vscode.ExtensionContext) {
     // NOTE: Fix `obtainWorksapceRoot` if non-null assertion is false
     const oneTreeDataProvider = new OneTreeDataProvider(this.workspaceRoot!);
-    context.subscriptions.push(
-        vscode.window.registerTreeDataProvider('OneExplorerView', oneTreeDataProvider));
 
-    const subscribeCommands = (disposals: vscode.Disposable[]) => {
+    // TODO Remove duplicated collapseAll command
```
Comment (jyoungyun) [inline src/OneExplorer/OneExplorer.ts:474]: (Q) Isn't `one.explorer.collapseAll` already removed?

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**